README.md

```diff
@@ -16,8 +16,27 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
-
+* [Use .on() for event binding](#use-on-for-event-binding)
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Use [.on()](http://api.jquery.com/on/) for event binding
+
+Methods like [`.click()`](http://api.jquery.com/click/) or [`.change()`](http://api.jquery.com/change/) are just alias for `.on('click')` and `.on('change')`. 
+
+### Why?
```

Comment:
"reduce memory usage", is that true? I would maybe add instead that if you don't use the aliases you can trim the jQuery build (wrote about that somewhere in issues / PR) as you can leave out the aliases. That way you reduce the file size. (and yes, I guess that reduces what has to be parsed and stored in memory).

Should we give an example of event delegation using `.on()` or is there a separate guideline we can refer to?